nrf_security/src/psa_crypto_driver_wrappers.c

```diff
@@ -2892,6 +2906,298 @@ psa_status_t psa_driver_wrapper_key_agreement(
     }
 }
 
+#if defined(CONFIG_PSA_CORE_OBERON)
+/*
+ * PAKE functions
+ */
+psa_status_t psa_driver_wrapper_pake_setup(
+    psa_pake_operation_t *operation,
+    const psa_pake_cipher_suite_t *cipher_suite)
+{
+    psa_status_t status;
+
```

Comment:
Nit: Since the pake  function are not standardized we are writing the functions in a different way here and we ifdef algorithms and not drivers. Maybe just write a small comment saying this.